README.md

```diff
@@ -61,4 +61,28 @@ client.on('messageCreate', message => {
         message.channel.createMessage(menu.build('Click this'));
     }
 });
-~~~
\ No newline at end of file
+~~~
+- Attachment
+~~~javascript
+const ErisAddons = require('eris-addons');
+const {Client} = require('eris');
+const client = new Client('bot token');
+
+client.on('messageCreate', message => {
+    if(message.content.toLowerCase() === '!attachment') {
+    const Attachment = new ErisAddons.Attachment()
+    Att.setFile(canvas.toBuffer())
+    Att.setName('file.png')
+
+     const embed = new ErisAddons.Embed();
+       embed.setAuthor(message.author.username, message.author.avatarURL);
+       embed.setTitle('Title of embed');
+       embed.setDescription('Description of embed');
+       embed.setColor('0x7289DA');
+       embed.setImage('attachment://file.png')
```

Comment:
about this... it creates a file in project like [jimp](https://www.npmjs.com/package/jimp)?
i had never seen this method

Reply:
This does not create a local file, this creates an attachment in the discord message itself, which can be accessed by `attachment://file.png`.

Reply:
i understand